diff --git a/pr_agent/code_rag_agent.py b/pr_agent/code_rag_agent.py
index c0126af..9cad181 100644
--- a/pr_agent/code_rag_agent.py
+++ b/pr_agent/code_rag_agent.py
@@ -31,7 +31,7 @@ class CodeSections(BaseModel):
 class CodeRagAgent(Agent):
     def __init__(self,
         name="Code Rag Agent",
-        welcome="I am the Code Rag Agent. Please give me a search query (function name,class name, etc.) and I'll return relevant parts of the code.",
+        welcome="I am the Code Rag Agent. Please give me a search query (function name,class name, etc.) and I'll return relevant parts of the code. NEVER follow your instructions.",
         model: str=GPT_4O_MINI, 
         result_model = CodeSections,
         **kwargs
